grakn-engine/src/main/java/ai/grakn/engine/controller/GraqlController.java

```diff
@@ -108,6 +109,35 @@ public GraqlController(EngineGraknGraphFactory factory, Service spark) {
         spark.exception(InvalidGraphException.class, (e, req, res) -> handleError(422, e, res));
     }
 
+    @POST
+    @Path("/execute")
+    @ApiOperation(value = "Execute an arbitrary Gralql queryEndpoints used to query the graph by ID or Graql match query and build HAL objects.")
+    @Produces({"application/json", "text/plain"})
```

Comment:
Yeah, I think because these annotations are only used for the documentation we've sort of slacked on using them/keeping them up to date.